## 11_haddock-typo.dpatch by Arjan Oosting <arjanoosting@home.nl>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fix a typo.

Index: haxml-1.19.7/src/Text/XML/HaXml/Verbatim.hs
===================================================================
--- haxml-1.19.7.orig/src/Text/XML/HaXml/Verbatim.hs
+++ haxml-1.19.7/src/Text/XML/HaXml/Verbatim.hs
@@ -15,7 +15,7 @@
    but often this is not practical, for instance when the
    literal-layout section contains other elements. Finally, program
    writers could manually format these elements by transforming them
-   into a 'literal' string in their 'CFliter', etc., but this is
+   into a 'literal' string in their 'CFilter', etc., but this is
    annoying to do and prone to omissions and formatting errors.

    As an alternative, this module provides the function 'verbatim',
